test/pk-src/copy-src.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+if [ ! -d riscv-pk ]; then
+  rsync --exclude ".git" -r ../../riscv-pk .
```

Comment:
For the sake of completeness,
```sh
mkdir -p riscv-pk
git -C ../../riscv-pk archive HEAD | tar -x -f - -C riscv-pk
```
is potentially faster if the working tree is dirtied by build objects, which `git archive` would avoid copying.
This matters less here but could be significant for the linux-src test.

Reply:
I'll change it if it becomes a problem. My way is easy to guess its function, and requires a single man-page lookup if it's still not clear. I don't even know what exactly the suggested command does and I doubt future readers would either. It also doesn't work when copy/pasted (I don't understand it enough to fix) and the ' - ' option is not easily discovered in the tar man page (I still don't know what it does, is it BASH magic?). Anyway, "premature optimization is the root of all evil" and all that.

Reply:
`git archive` creates a tarball from the specified revision, similar to GitHub releases.
`-` is the argument to `-f` and, by Unix convention, is shorthand for stdin.